environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   # Packages required for setting up the environment
   - pip>=21,<24
-  - python>=3.10,<3.12
+  - python>=3.10,<=3.12
```

Comment:
Because I'm now using the PUDL Name Cleaner and have PUDL as a dependency, I needed to upgrade the python version